### DISCUSSION TITLE
Need help with using glads multi context GL with glads wgl with win32 API

### DISCUSSION THREAD

**Discussion by A1D2D** [Dav1dde/glad#482] [Q&A]
I have created a simple window in win32 i downloaded a multi context gl and wgl package from glad to add graphics interface however if I use the multi context version i get a error when at glad_gl_get_proc_from_userptr. But i found something wierder I copyed the wrapping from the gladLoadGLContext function and it somehow worked so i only can load opengl contexts trough using  gladLoadGLContextUserPtr, but gladLoadGLContext, gladLoaderLoadGLContext fails. Pls help me figure out what might be the issue and maybe a example project on github might be beneficial.

**Comment by Dav1dde** [comment]
Can you show your code, the exact error you're getting and the include the header of `glad/{wgl,gl}.h` which shows what you generated?

This header:

```
 * APIs:
 *  - gl:compatibility=4.6
 *
 * Options:
 *  - ALIAS = False
 *  - DEBUG = False
 *  - HEADER_ONLY = False
 *  - LOADER = False
 *  - MX = False
 *  - ON_DEMAND = False
 *
 * Commandline:
 *    --api='gl:compatibility=4.6' --extensions='' c
```

**Reply by A1D2D** [reply]
```
* Generator: C/C++
 * Specification: wgl
 * Extensions: 57
 *
 * APIs:
 *  - wgl=1.0
 *
 * Options:
 *  - ALIAS = True
 *  - DEBUG = False
 *  - HEADER_ONLY = False
 *  - LOADER = True
 *  - MX = True
 *  - ON_DEMAND = False
```

```
 * Generator: C/C++
 * Specification: gl
 * Extensions: 836
 *
 * APIs:
 *  - gl:compatibility=4.6
 *  - gles2=3.2
 *
 * Options:
 *  - ALIAS = True
 *  - DEBUG = False
 *  - HEADER_ONLY = False
 *  - LOADER = True
 *  - MX = True
 *  - ON_DEMAND = False
```

Btw i just solved the problem with "my own" get_proces function: void* 
```
get_proc(const char* name) {
   void* p = (void*)wglGetProcAddress(name);
   if(p == 0 ||
      (p == (void*)0x1) || (p == (void*)0x2) || (p == (void*)0x3) ||
      (p == (void*)-1) )
   {
      HMODULE module = LoadLibraryA("opengl32.dll");
      p = (void*)GetProcAddress(module, name);
   }
   return p;
}
``` 
well for my code i esentaly tried using the wgl example provided by glad but with multi context. But now somehow (idk how) i made gladLoaderLoadGLContext work to so i might just messed up some includes or something

**Reply by A1D2D** [reply]
But still i do not have a clear idea of how should i get a GLADloadfunc using library defined functions? ,because my method is kinda rubbish.
